api/src/main/java/org/openmrs/Concept.java

```diff
@@ -186,8 +186,12 @@ public Concept(Integer conceptId) {
 	 * 
 	 * @param answers The answers to set.
 	 */
+
 	public void setAnswers(Collection<ConceptAnswer> answers) {
-		this.answers = answers;
+		this.answers.clear();
```

Comment:
I do not want to add a clear() + addAll() just to this one hibernate-managed collection across all of OpenMRS.

This might be the right thing to do, but if so we should make an effort to do it consistently across the whole codebase.

I made a comment here: https://talk.openmrs.org/t/trunk-5144-concept-save-for-coded-concept-throws-hibernate-exception/11059/3

I think there are two easy alternatives we can do now:

option 1: change in the legacyui module to just use getAnswers() instead of getAnswers(true).

option 2: in core, in getAnswers(boolean), do if (include == true) return getAnswers().